zephyr-sync-report-cucumber/src/main/java/lv/ctco/zephyr/transformer/CucumberTransformer.java

```diff
@@ -58,7 +58,7 @@ String readCucumberReport(String path) {
                 test.setUniqueId(generateUniqueId(feature, scenario));
                 test.setLabels(extractTagValues(scenario, "@Labels="));
                 test.setStoryKeys(extractTagValues(scenario, "@Stories="));
-                test.setDescription(extractTagValue(scenario, "@Description="));
+                test.setDescription(scenario.getDescription() == null ? (feature.getDescription() == null ? "" :  feature.getDescription()): scenario.getDescription());
```

Comment:
not very well readable, can you please extract to a separate private method like "extractDescription"?